idaes/core/unit_model.py

```diff
@@ -656,5 +657,13 @@ def initialize(blk, state_args=None, outlvl=idaeslog.NOTSET,
         # Release Inlet state
         blk.control_volume.release_state(flags, outlvl)
 
+        if (results is not None and (
```

Comment:
Question: under what conditions would results be `None`?

Reply:
Looks like this got accidentally included here by copying from property package (where there are cases where results is `None`).